.travis.yml

```diff
@@ -16,11 +15,6 @@ cache:
 
 sudo: true
 
-addons:
-  apt:
-    packages:
-      - docker-ce
```

Comment:
Is this not needed now?

Reply:
Yes there is some build issue earlier we had introduced this I have upgraded the golang version and travis to latest